applications/GeoMechanicsApplication/tests/cpp_tests/test_utilities/geo_custom_element.cpp

```diff
@@ -0,0 +1,94 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Aron Noordam
+//
+#include "custom_utilities/dof_utilities.h"
```

Comment:
Could we just include dof.h instead of dof_utilities? or are we using something from the utils?

Reply:
will check